js/modules/backup.js

```diff
@@ -453,7 +455,7 @@ function _getAnonymousAttachmentFileName(message, index) {
 
 async function readAttachment(dir, attachment, name, options) {
   options = options || {};
-  const { key, encrypted } = options;
```

Comment:
Simplified this to have the following implication `Boolean(key) ==> encrypted === true`.